app/pages/Accounts/TransactionView/TransactionView.module.scss

```
@@ -65,3 +65,35 @@
     top: 9px;
     right: 12px;
 }
+
+.eventsButtonOpen,
+.eventsButtonClosed {
+    padding: 0;
+}
+
+.eventsButtonOpen {
+    svg {
+        transform: rotate(90deg);
+    }
+}
+
+.eventsButtonClosed {
+    svg {
+        transform: rotate(-90deg);
+    }
+}
```

Would suggest the following changes, which will:

- Align with other icon buttons in details view (bg with padding on hover)
- Invert arrows (you'd expect the icon to point downwards when closed, and vice versa)

```suggestion
.eventsButtonOpen,
.eventsButtonClosed {
    position: absolute;
    right: 14px;
    padding-right: 0;
}

.eventsButtonOpen {
    transform: rotate(-90deg);
}

.eventsButtonClosed {
    transform: rotate(90deg);
}
```

---

Great suggestion (Neither I nor Jakob noticed the arrows were wrong)